resolwe_bio/processes/workflows/wgbs.yml

```diff
@@ -61,6 +61,180 @@
             Merging CpG pairs results in symmetric methylation levels. Methylation is usually
             symmetric (cytosines at CpG sites were methylated on both DNA strands). Choosing this
             option will only keep the CpG sites data.
+    - name: summary
+      label: Picard AlignmentSummary
+      group:
+        - name: adapters
+          label: Adapter sequences
+          type: data:seq:nucleotide
+          required: false
+        - name: validation_stringency
+          label: Validation stringency
+          type: basic:string
+          choices:
+            - label: STRICT
+              value: STRICT
+            - label: LENIENT
+              value: LENIENT
+            - label: SILENT
+              value: SILENT
+          default: STRICT
+          description: |
+            Validation stringency for all SAM files read by this
+            program. Setting stringency to SILENT can improve
+            performance when processing a BAM file in which
+            variable-length data (read, qualities, tags) do not
+            otherwise need to be decoded. Default is STRICT.
+        - name: insert_size
+          label: Maximum insert size
+          type: basic:integer
+          default: 100000
+        - name: pair_orientation
+          label: Pair orientation
+          type: basic:string
+          choices:
+            - label: Unspecified
+              value: 'null'
+            - label: FR
+              value: FR
+            - label: RF
+              value: RF
+            - label: TANDEM
+              value: TANDEM
+          default: 'null'
+    - name: insert
```

Comment:
This block of inputs is not used, as `InsertSizeMetrics` is not used in the workflow.